pom.xml

```diff
@@ -400,10 +394,25 @@
     <dependency>
       <groupId>org.jenkins-ci.plugins</groupId>
       <artifactId>junit</artifactId>
-      <version>1.3</version>
+      <version>1.20</version>
+      <scope>test</scope>
     </dependency>
   </dependencies>
 
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+        <groupId>org.jenkins-ci.main</groupId>
+        <artifactId>jenkins-test-harness</artifactId>
```

Comment:
Do we need this if this is part of https://github.com/jenkinsci/plugin-pom/blob/master/pom.xml#L59 ?

Reply:
this is fixing enforcer upperBoundDependency